### DISCUSSION TITLE
Error on Netlify

### DISCUSSION THREAD

**Discussion by twlite** [nextauthjs/next-auth#2044] [Help]
## Question 💬

In development, everything's fine. I use netlify to host my website. When I try to login, it throws error:
![image](https://user-images.githubusercontent.com/46562212/119219208-0c909f80-bb04-11eb-9309-c84fb0cb95a9.png)

I tried the same code in vercel, where the login works without any problem but in Sign In page, the `Sign in with Discord` redirects to the same page of `deployment preview` rather than the main website.
![image](https://user-images.githubusercontent.com/46562212/119219231-3a75e400-bb04-11eb-9a4a-8a5cdc264a69.png)

# Env
```
CLIENT_ID=DISCORD_CLIENT_ID
CLIENT_SECRET=DISCORD_CLIENT_SECRET
NEXTAUTH_URL=https://snowflakedev.xyz <--- tried adding this but nothing happened
SESSION_SECRET=keyboard-cat
```

**Comment by twlite** [comment]
@balazsorban44 any update on this? since you moved the issue here

**Reply by balazsorban44** [reply]
Moved this here because it was a question not a bug report, with little to no reproduction. This is our current flow so we can keep focusing on reports that are potentially/actually bugs feature requests. Everything else is seen as user/configuration error until proved otherwise. I don't use Netlify, so a hosted version plus a reproduction repo would help a lot here.

**Reply by twlite** [reply]
Sure, here is the repo & the live site..
**Repo**: https://github.com/DevSnowflake/website
**Live website**: https://snowflakedev.xyz

**Comment by zyrouge** [comment]
Having the same issue. Any updates?
